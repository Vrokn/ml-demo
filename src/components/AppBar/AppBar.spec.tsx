import React from 'react';
import { render } from '@testing-library/react';
import DrawerAppBar from './AppBar';
import { BrowserRouter } from 'react-router-dom';

describe('DrawerAppBar', () => {
  it('matches snapshot', () => {
    const { asFragment } = render(
      <BrowserRouter>
        <DrawerAppBar />
      </BrowserRouter>
    );
    expect(asFragment()).toMatchSnapshot();
  });
});
