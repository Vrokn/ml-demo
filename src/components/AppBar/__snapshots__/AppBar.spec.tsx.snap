// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`DrawerAppBar matches snapshot 1`] = `
<DocumentFragment>
  <header
    class="MuiPaper-root MuiPaper-elevation MuiPaper-elevation4 MuiAppBar-root MuiAppBar-colorInherit MuiAppBar-positionStatic css-1cjrmu8-MuiPaper-root-MuiAppBar-root"
  >
    <div
      class="MuiToolbar-root MuiToolbar-gutters MuiToolbar-regular css-p7xqs8-MuiToolbar-root"
    >
      <a
        href="/"
      >
        <img
          alt="logo"
          height="50px"
          src="https://www.max.live/hs-fs/hubfs/black-lockup-72ppi-1.png?width=250&height=93&name=black-lockup-72ppi-1.png"
        />
      </a>
      <div
        class="MuiBox-root css-9aj5lt"
      >
        <a
          class="MuiButtonBase-root MuiButton-root MuiButton-outlined MuiButton-outlinedPrimary MuiButton-sizeMedium MuiButton-outlinedSizeMedium MuiButton-root MuiButton-outlined MuiButton-outlinedPrimary MuiButton-sizeMedium MuiButton-outlinedSizeMedium css-1rwt2y5-MuiButtonBase-root-MuiButton-root"
          href="/"
          tabindex="0"
        >
          Search
          <span
            class="MuiTouchRipple-root css-8je8zh-MuiTouchRipple-root"
          />
        </a>
        <a
          class="MuiButtonBase-root MuiButton-root MuiButton-outlined MuiButton-outlinedPrimary MuiButton-sizeMedium MuiButton-outlinedSizeMedium MuiButton-root MuiButton-outlined MuiButton-outlinedPrimary MuiButton-sizeMedium MuiButton-outlinedSizeMedium css-1rwt2y5-MuiButtonBase-root-MuiButton-root"
          href="/favorites"
          tabindex="0"
        >
          Favorites
          <span
            class="MuiTouchRipple-root css-8je8zh-MuiTouchRipple-root"
          />
        </a>
      </div>
    </div>
  </header>
</DocumentFragment>
`;
